doc_aliases.py

```diff
@@ -0,0 +1,184 @@
+import os
+import sys
+
+
+IGNORE_C_FNS = ["_get_reference_count", "_status", "_free", "_destroy", "_unref", "gdk_device_free_history"]
+IGNORE_FNS = [
+    "deref", "deref_mut", "drop", "fmt", "clone", "to_value", "g_value_get_flags",
+    "g_value_set_flags", "get_type",
+]
+IGNORE_FNS_START = ["to_glib", "from_glib", "from_raw", "to_raw"]
```

Comment:
```suggestion
IGNORE_FNS_START = ["into_glib", "from_glib", "from_raw", "to_raw"]
```
we don't have `to_glib` anymore

Reply:
We have it in `cairo`. ;) Gonna add `into_glib` though.